src/backend/distributed/utils/enable_ssl.c

```diff
@@ -37,7 +37,17 @@
 #define X509_SUBJECT_COMMON_NAME "CN"
 
 #define POSTGRES_DEFAULT_SSL_CIPHERS "HIGH:MEDIUM:+3DES:!aNULL"
-#define CITUS_DEFAULT_SSL_CIPHERS "TLSv1.2+HIGH:!aNULL:!eNULL"
+#define CITUS_DEFAULT_SSL_CIPHERS_OLD "TLSv1.2+HIGH:!aNULL:!eNULL"
+
+/* Microsoft approved cipher string */
```

Comment:
leave comment here to indicate this implicitly enforces TLSv1.2

Reply:
Updated te comment with more info